src/Routing/Route.php

```diff
@@ -2,61 +2,40 @@
 
 namespace Tnapf\Router\Routing;
 
-use Closure;
 use InvalidArgumentException;
-use ReflectionMethod;
 use stdClass;
 use Tnapf\Router\Enums\Methods;
 use Tnapf\Router\Router;
+use Tnapf\Router\Interfaces\RequestHandlerInterface;
 
 class Route {
     public readonly string $uri;
     private array $methods;
-    private array $before = [];
-    private array $after = [];
+    private array $middleware = [];
+    private array $postware = [];
     private stdClass $parameters;
 
    /**
     * @param string $uri
```

Comment:
**Question**
If you dont mind me asking, why do you add doc comments for properties that are completely supported as strict type. Thus an IDE already knows how to typehint it?

It doesn't really matter in theory, it's just easy to forget updating comments when you update the code.

Reply:
No real reason, it's a really inconsistent habit tbh